Makefile

```
@@ -154,12 +159,17 @@ image-build-%: check-container-tool ## Build Docker image ## Build Docker image
  		-t $($*_IMAGE) -f Dockerfile.$* .
 
 .PHONY: image-push
-image-push: image-push-epp image-push-sidecar ## Push Docker images to registry
+image-push: epp-image-push sidecar-image-push ## Push container images to registry
```

You accidentally undone an important fix on this line.
```suggestion
image-push: image-push-epp image-push-sidecar ## Push container images to registry
```